packages/block-editor/src/hooks/color.scss

```diff
@@ -0,0 +1,33 @@
+.block-editor__hooks-colors-panel {
+	.components-dropdown {
+		display: block;
+	}
+
+	// Allow horizontal overflow so the size-increasing color indicators don't cause a scrollbar.
+	.components-navigator-screen {
+		overflow-x: visible;
```

Comment:
`NavigatorScreen` already has a `overflow-x: 'auto'`, why do we need to force it to `visible` ?

Reply:
That rule can be removed. I suspect it's a remnant as this PR was forked from the one that included the navigator. [I didn't feel very comfortable with targetting those classes either](https://github.com/WordPress/gutenberg/pull/36952#issuecomment-983570048).